packages/block-editor/src/components/off-canvas-editor/appender.js

```diff
@@ -52,20 +52,10 @@ export const Appender = forwardRef( ( props, ref ) => {
 	let maybeLinkUI;
 
 	if ( insertedBlock ) {
-		const link = {
-			url: insertedBlockAttributes.url,
-			opensInNewTab: insertedBlockAttributes.opensInNewTab,
-			title: insertedBlockAttributes.label,
-		};
```

Comment:
Previously you needed to know about the structure of this object. Now you just pass all the attributes.